code/modules/reagents/reagent_containers/jerrycan.dm

```diff
@@ -0,0 +1,178 @@
+#define LABEL_TEXT "text"
+#define LABEL_TEXT_OLD "old"
+#define LABEL_INFLAMMABLE "inflammable"
+#define LABEL_NT "nt"
+#define LABEL_NT_MINI "nt_mini"
+#define LABEL_SUSPICIOUS "sus"
+#define LABEL_SUSPICIOUS_BLACK "sus_black"
+#define LABEL_SUSPICIOUS_MINI "sus_mini"
+#define LABEL_SUSPICIOUS_MINI_BLACK "sus_mini_black"
+#define LABEL_EZ_NUTRIENT "ez"
+#define LABEL_ROBUST_HARVEST "robust"
+#define LABEL_LEFT_4_ZED "l4z"
+#define LABEL_SPACE_CLEANER "space_cleaner"
+
+#define CAP_BLACK "black"
+#define CAP_WHITE "white"
+#define CAP_RED "red"
+#define CAP_GREEN "green"
+#define CAP_BLUE "blue"
```

Comment:
What are these for? a few are totally unused and I think only 3 are ever used more then once, why are these defines?

Reply:
I added these defines to improve extendability, the thought was it would help future coders choose label and cap variants by providing them with a handy list in vscode without having to open dm.

I expect other coders would potentially add lots of new types of cans and this would help facilitate that.

Spriting these kinds of small labels can be more tricky than it would seem so i decided to include a few extra i forsee people will want in use later on. 

Of course, unused sprites should probably be in a separate repo to not create digital litter in the main repo, but very few people know about the unused sprite repo and i feel these will see use quite soon.

Reply:
I agree these should stay in repo, tho these defines do kind of put me off. it's not a huge deal, they just strike me as weird instead of helpful